README.md

```diff
@@ -41,56 +42,58 @@ OpenChatML is intended to be used in various conversational AI scenarios, includ
 4. Human-in-the-Loop Interactions: Representing and capturing interactions between human agents and AI systems for training and evaluation.
 
 ## 2. Tokens
+
 OpenChatML uses the following special tokens:
 
-- `<s>`: Beginning of Sequence (BOS) token, indicating the start of a conversation.
-- `</s>`: End of Sequence (EOS) token, indicating the end of the conversation.
-- `<|im_start|>`: Start of Turn token, indicating the beginning of a new message within the conversation. 
+- `<|BOS|>`: Beginning of Sequence (BOS) token, indicating the start of a conversation.
+- `<|EOS|>`: End of Sequence (EOS) token, indicating the end of the conversation.
+- `<|im_start|>`: Start of Turn token, indicating the beginning of a new message within the conversation.
 - `<|im_end|>`: End of Turn token, indicating the end of the current message.
 - `<|fim_prefix|>`: Before Cursor token, indicating the content before the cursor in a fill-in-the-middle task.
 - `<|fim_suffix|>`: After Cursor token, indicating the content after the cursor in a fill-in-the-middle task.
 - `<|fim_middle|>`: At Cursor token, indicating where the model should fill in content in a fill-in-the-middle task.
 - `<|file_separator|>`: File Separator token, used to separate content from different files within the same sequence.
-- `<|startofthought|>`: Start of Thought token, indicating the beginning of a thought or rationale.
-- `<|endofthought|>`: End of Thought token, indicating the end of a thought or rationale.
+- `<|start_of|>thought`: Start of Thought token, indicating the beginning of a thought or rationale.
+- `<|end_of|>thought`: End of Thought token, indicating the end of a thought or rationale.
 
 ## 3. Message Structure
+
 Each message in OpenChatML is represented as follows:
 
 ```
-<|im_start|>role [name=<name>]
-message_content
-<|im_end|>
+<|im_start|>role [<name>]
```

Comment:
this kind of change won't be accepted without a very strong and compelling argument.